scripts/gulpfiles/build_tasks.js

```diff
@@ -173,7 +173,7 @@ function stripApacheLicense() {
  * https://github.com/google/closure-compiler/blob/master/src/com/google/javascript/jscomp/DiagnosticGroups.java#L113
  */
 var JSCOMP_ERROR = [
-  'accessControls',
+  // 'accessControls',
```

Comment:
It turns out that this is now just a synonym for `visibility`, so it can just be deleted.

Reply:
Done!